docs/api.md

```diff
@@ -752,7 +752,16 @@ Parameter     | Type   | Optional | Default | Description
 ---------     | ----   | -------- | ------- | -----------
 iterations    | int    | yes      | N/A     | Max number of solver's iterations
 snapshot      | int    | yes      | N/A     | Iterations between model snapshots
-solver_type   | string | yes      | SGD     | from "SGD", "ADAGRAD",  "RMSPROP", "ADAM"
+solver_type   | string | yes      | SGD     | from "SGD", "ADAGRAD",  "RMSPROP", "ADAM", "RANGER"
```

Comment:
we may want RANGER_PLUS to make the API identical to that of https://github.com/jolibrain/deepdetect/pull/901

Reply:
ok done